pyogrio/geopandas.py

```diff
@@ -115,6 +115,11 @@ def read_dataframe(
     fid_as_index : bool, optional (default: False)
         If True, will use the FIDs of the features that were read as the
         index of the GeoDataFrame.  May start at 0 or 1 depending on the driver.
+    use_arrow : bool, default False
+        Whether to use Arrow as the transfer mechanism of the read data
+        from GDAL to Python (requires GDAL >= 3.6 and `pyarrow` to be
+        installed). When enabled, this provides a further speed-up.
+        Currently not enabled by default.
```

Comment:
```suggestion

```
(no need to repeat the default value above unless this is intended to indicate _current_ behavior that will be changed in the future)